scipy/stats/_morestats.py

```diff
@@ -1831,9 +1832,9 @@ def shapiro(x):
 _Avals_logistic = array([0.426, 0.563, 0.660, 0.769, 0.906, 1.010])
 
 
-AndersonResult = namedtuple('AndersonResult', ('statistic',
-                                               'critical_values',
-                                               'significance_level'))
+AndersonResult = _make_tuple_bunch('AndersonResult',
+                                   ['statistic', 'critical_values',
+                                    'significance_level'], ['fit_result'])
```

Comment:
Should we add `pvalue` as an alias for `significance_level` as we did in other places?

Reply:
Currently the does not output a single p-value; rather, it outputs tables of critical values and associated significance levels.  `anderson_ksamp` does linear interpolation. We could do that here. But really, I would like to use `monte_carlo_test` to produce a more accurate p-value. Please see items 3 and 4 at https://github.com/scipy/scipy/issues/10829#issuecomment-1179351989. I'd be happy to write it (separate PR) if you'd be willing to review.

Reply:
ok I see, thanks for the details. If you say that it MC here be done in a reasonable time, why not. I am happy to help review.